internal/sdkprovider/service/grafana/grafana_test.go

```diff
@@ -695,3 +696,81 @@ resource "aiven_grafana" "bar" {
 }
 `, networks, os.Getenv("AIVEN_PROJECT_NAME"), name)
 }
+
+// TestAccAiven_grafana_and_default_vpc
+// Tests that when `project_vpc_id` is not set, the service is NOT created in the default VPC.
+// The backend automatically creates a service in the default VPC,
+// so terraform must send nil in `project_vpc_id`.
+// Tests all service types because the logic is in the shared code, but grafana runs faster.
```

Comment:
nit: I'm having a hard time following this comment

suggestion: 


```suggestion
// Verifies that when `project_vpc_id` is not set, the service is NOT created in the default VPC.
// The backend automatically assigns a service to the default VPC if `project_vpc_id` is not specified,
// so terraform must ensure `project_vpc_id` is explicitly set to nil.
// Although this behavior affects all service types due to shared backend logic, we use Grafana for testing as it has the fastest setup time.
```